ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -48,6 +50,8 @@ internal class PaywallViewModelImpl(
     applicationContext: ApplicationContext,
     private val offering: Offering?,
     private val listener: PaywallViewListener?,
+    private val packageName: String,
```

Comment:
An alternative to passing the package name here would be to add a method in the `ApplicationContext` to get it... I'm ok keeping it as a separate parameter though if you prefer it.

Reply:
I agree 👍🏻 

Also I know this matches the Android property, but could we change this so it doesn't get confused with `RevenueCat.Package`? It's kinda confusing now hehe. Maybe `applicationManifestName`?

Reply:
I named it `applicationPackageName`, I think it's less confusing. I find `applicationManifestName` strange 🤷